{{cookiecutter.project_slug}}/.pre-commit-config.yaml

```diff
@@ -111,7 +111,7 @@ repos:
     {% if cookiecutter.use_mccabe in ['yes', 'y'] -%}
     -   id: mccabe
         name: mccabe
-        entry: mccabe --max-complexity=5
+        entry: python -m mccabe --min 5 *.py
```

Comment:
@xmnlab Do you have any idea why this is not working?

Reply:
this is what I found in the documentation `python -m mccabe --min 5 mccabe.py` ... so it should work .. what is the error message you have?

Reply:
Executable `mccabe` not found

https://github.com/osl-incubator/cookiecutter-python/actions/runs/5075606779/jobs/9116994061#step:5:369

Reply:
not sure tbh .. I can give you a hand on that tomorrow

Reply:
A thousand thanks!

For now, I implemented it with the flake8 plugin and it worked.